src/components/NewCommentForm.tsx

```diff
@@ -75,25 +183,43 @@ export const NewCommentForm: React.FC = () => {
             id="comment-body"
             name="body"
```

Comment:
as we have name attribute body for comment - we need to make instead comment field `body` field for newComment and error state. or rename attribute here.